src/components/SubmitTokenInfo/index.tsx

```diff
@@ -0,0 +1,250 @@
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Tooltip from 'antd/lib/tooltip'
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { utils } from '@ckb-lumos/base'
+import CloseIcon from '../../assets/modal_close.png'
+import HelpIcon from '../../assets/qa_help.png'
+import AlertIcon from '../../assets/alert.png'
+import { LabeledInput } from './LabeledInput'
+import styles from './styles.module.scss'
+import { ImgUpload } from './ImgUpload'
+import CommonButton from '../CommonButton'
+import CommonModal from '../CommonModal'
+import { submitTokenInfo } from '../../services/ExplorerService/fetcher'
+import CommonSelect from '../CommonSelect'
+import { isMainnet } from '../../utils/chain'
+import { MainnetContractHashTags, TestnetContractHashTags } from '../../constants/scripts'
+
+type Props = {
+  isOpen: boolean
+  onClose: () => void
+}
+
+const LabelTooltip = ({ title, icon }: { title: string; icon?: string }) => (
+  <Tooltip placement="bottom" title={title}>
+    <img src={icon ?? HelpIcon} alt="tooltip" className={styles.tooltipIcon} />
+  </Tooltip>
+)
+
+export const SubmitTokenInfo = (prop: Props) => {
+  const { t } = useTranslation()
+  const { onClose, isOpen } = prop
+  const [args, setArgs] = useState('')
+
+  const [symbol, setSymbol] = useState('')
+  const [name, setName] = useState('')
+  const [decimal, setDecimal] = useState('')
+  const [description, setDescription] = useState('')
+  const [website, setWebsite] = useState('')
+  const [creatorEmail, setCreatorEmail] = useState('')
+  const [logo, setLogo] = useState<string | null>(null)
+
+  const scriptDataList = isMainnet() ? MainnetContractHashTags : TestnetContractHashTags
+  const tokenTypeOptions = scriptDataList
+    .filter(scriptData => scriptData.tag.includes('sudt'))
+    .reverse() // sUDT should come first, then sUDT(deprecated)
```

Comment:
The order should be defined by logic(`Array.prototype.sort`) if there's a specific rule

Ref:
Array.prototype.sort: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort